db/seeds/questionnaires/mentor_diary.rb

```diff
@@ -125,28 +125,35 @@
   labels: ['niet', 'heel sterk'],
   section_end: true
```

Comment:
`section_end` hier moet weg. Die staat nu bij v8.

Het aantal `section_start`s en `section_end`s in een vragenlijst moet altijd overeenkomen.